### PR TITLE
chess 1.0.0

### PR DESCRIPTION
**Pull request by MrAnt1vit**
run main.py in ./code

### CODE REVIEW

**Comment by MrAnt1vit1147** [comment]
jhg

**Comment by MrAnt1vit** [comment]
ghf

**Comment by MrAnt1vit1147** [comment]
fdg

File: code/chess/bishop.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+from chess.figure import Figure
+
+
+class Bishop(Figure):
+    """ Слон """
+
+    def score(self) -> int:
```
Comment (MrAnt1vit1147) [inline code/chess/bishop.py:7]: jhgjh

**Review by MrAnt1vit1147 — COMMENTED** [review]
kk

**Pull request closed without merging**